django/utils/exception_handler.py

```diff
@@ -0,0 +1,15 @@
+from django.http.response import JsonResponse
+from rest_framework.status import HTTP_500_INTERNAL_SERVER_ERROR
+from rest_framework.views import exception_handler
+
+
+def custom_exception_handler(exc, context):
+    # Call REST framework's default exception handler first,
+    # to get the standard error response.
+    response = exception_handler(exc, context)
+
+    return JsonResponse(
+        str(exc),
+        status=response.status_code if response is not None else HTTP_500_INTERNAL_SERVER_ERROR,
```

Comment:
when is `response` None ?

Reply:
Django `exception_handler` can return None apparently. This raises an error in my linter because of this